
{'date': 'Fri Dec 10 2021 13:29:33.421 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
RuntimeError: too much recursion
  module '$exec_2098' line 27
        self.vai()
'''},
{'date': 'Fri Dec 10 2021 13:29:53.946 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 31
    Gatil(GATIL_MOS).vai()
  module <module> line 28
    c0 = Elemento(img, x=140, y=340, w=200, h=200, cena=self)
NameError: name 'img' is not defined
'''},
{'date': 'Thu Feb 03 2022 18:45:31.150 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 45
    class Sala(SalaVit):
  module <module> line 46
    def __init__(self, n=NADA, l=NADA, s=NADA, o=NADA, nome='', **kwargs):
NameError: name 'NADA' is not defined
'''},
{'date': 'Fri Feb 04 2022 08:39:05.484 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 40
  class Hero(Elemento)
                       ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Feb 04 2022 08:40:01.86 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 47
  self.props ={P.H: Hero, P.T}
                              ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Feb 04 2022 08:40:39.545 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: bases[i].$dict is undefined
  module '$exec_339' line 5
class P(Enum):
'''},
{'date': 'Fri Feb 04 2022 09:40:04.520 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 48
  super().__init__(IMP.format(STRAY[0], x=x, y=y, w=w, h=h, cena=cena)
                                                                                ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Feb 04 2022 13:52:09.674 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 67
  self.nome = nome
            ^
SyntaxError: keyword can't be an expression
'''},
{'date': 'Fri Feb 04 2022 13:59:08.555 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 119
    Gatil(GATIL_MOS).vai()
  module <module> line 113
    et = Elemento(GITRAW, x=500, y=200, w=100, h=100, cena=sala_b.norte)
NameError: name 'GITRAW' is not defined
'''},
{'date': 'Fri Feb 04 2022 14:00:31.320 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 119
    Gatil(GATIL_MOS).vai()
  module <module> line 113
    et = Elemento(GITRAW, x=500, y=200, w=100, h=100, cena=sala_b.norte)
NameError: name 'GITRAW' is not defined
'''},
{'date': 'Fri Feb 04 2022 14:22:13.876 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 123
    Gatil(GATIL_MOS).vai()
  module <module> line 117
    document['g1111'].bind('click', self.et_vai)
KeyError: g1111
'''},
{'date': 'Fri Feb 04 2022 14:41:57.446 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 67
  s = svg.svg( width="300" height="300")
                            ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Feb 04 2022 14:49:51.339 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 128
    Gatil(GATIL_MOS).vai()
  module <module> line 125
    Thrash(sala_b.norte.elt)
  module <module> line 67
    s = svg.svg(xmlns="http://www.w3.org/2000/svg", version="1.1", viewBox="0 0 400 400", width="300", height="300")
ValueError: can't set attribute xmlns
'''},
{'date': 'Sat Feb 05 2022 11:46:52.649 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 147
    Gatil(GATIL_MOS).vai()
  module <module> line 145
    Thrash(x.elt, sala_b.norte)
  module <module> line 67
    cache = create_script_tag(GITRAW)
  module <module> line 87
    _tag = document.createElement('svg')
NameError: name 'document' is not defined
'''},
{'date': 'Sat Feb 05 2022 11:49:26.299 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: $B.get_class(...) is undefined
  module '$exec_320' line 148
Gatil(GATIL_MOS).vai()
'''},
{'date': 'Sat Feb 05 2022 11:51:58.772 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: $B.get_class(...) is undefined
  module '$exec_320' line 148
Gatil(GATIL_MOS).vai()
'''},
{'date': 'Mon Feb 07 2022 12:34:14.691 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 176
    Gatil(GATIL_MOS).vai()
  module <module> line 174
    Thrash(x.elt, sala_b.norte.elt)
  module <module> line 104
    s <= circle
NameError: name 'circle' is not defined
'''},
{'date': 'Mon Feb 07 2022 12:43:38.133 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 103
  obj = svg.use(href=f"#{label}" x= 50*dx, y=50*dy,
                                  ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Feb 07 2022 16:18:30.125 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 185
    Gatil(GATIL_MOS).vai()
  module <module> line 183
    Thrash(x.elt, sala_b.norte.elt)
  module <module> line 85
    cache = self.create_script_tag(LIXAO)
  module <module> line 122
    _fp = urllib.request.urlopen(src)
  module urllib.request line 37
    raise error. HTTPError('file not found')
HTTPError: file not found
'''},
{'date': 'Mon Feb 07 2022 17:57:51.66 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 187
    Gatil(GATIL_MOS).vai()
  module <module> line 185
    Thrash(x.elt, sala_b.norte.elt)
  module <module> line 111
    obj = svg.use(href=f"#{label}", x=5000 - 400*randint(0,9), y=300 -r200*randint(0,9), width=50, height=50,
NameError: name 'r200' is not defined
'''},
{'date': 'Wed Feb 09 2022 10:44:03.130 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 129
  def vai(self, ev):f"translate({dx} {dy})  rotate({7*indice} {ROFFX} {ROFFY}) scale(2.5)")
                      ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Feb 09 2022 18:13:23.270 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 139
  food = Elemento('', x=0, y=0, w=200, h=200 cena=self.cena)
                                              ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Feb 09 2022 19:21:59.982 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 200
    Gatil(GATIL_MOS).vai()
  module <module> line 198
    Thrash(x.elt, sala_b.norte)
  module <module> line 93
    elt <= s
NameError: name 's' is not defined
'''},
{'date': 'Wed Feb 09 2022 20:07:34.750 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 92
  def dump(self, cena, sorte=4)
                                ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Feb 09 2022 20:15:37.933 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 208
    Gatil(GATIL_MOS).vai()
  module <module> line 164
    self.trash = Thrash(x.elt, sala_b.norte)
AttributeError: 'int' object has no attribute 'elt'
'''},
{'date': 'Wed Feb 09 2022 20:17:26.282 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 209
    Gatil(GATIL_MOS).vai()
  module <module> line 178
    sala_a = Sala(*[IM.format(lnk) for lnk in SA])
  module <module> line 155
    self.cenas = [Rua(img) if isinstance(img, str) else img for img in [n, l, s, o]]
TypeError: __init__ missing 1 positional argument: 'trash'
'''},
{'date': 'Wed Feb 09 2022 20:34:39.752 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: $B.get_class(...) is undefined
  module '$exec_36683' line 208
Gatil(GATIL_MOS).vai()
'''},
{'date': 'Wed Feb 09 2022 20:53:07.955 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 92
  def dump(self, cena, sorte=4)
                                ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Feb 09 2022 21:01:47.10 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 208
    Gatil(GATIL_MOS).vai()
  module <module> line 182
    sala_b_args[0] = Rua(sala_b_args[0], self.trash, [
  module <module> line 78
    [p[proname](*proargs) for proname, proargs  in props]
  module <module> line 66
    super().__init__(HALO, x=x, y=y, w=w, h=h, o=0.2, vai=self.cena, cena=cena)
AttributeError: 'Trash' object has no attribute 'cena'
'''},
{'date': 'Wed Feb 09 2022 21:02:01.783 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 208
    Gatil(GATIL_MOS).vai()
  module <module> line 182
    sala_b_args[0] = Rua(sala_b_args[0], self.trash, [
  module <module> line 78
    [p[proname](*proargs) for proname, proargs  in props]
  module <module> line 66
    super().__init__(HALO, x=x, y=y, w=w, h=h, o=0.2, vai=self.cena, cena=cena)
AttributeError: 'Trash' object has no attribute 'cena'
'''},
{'date': 'Wed Feb 09 2022 21:47:17.12 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 2
    from _spy.vitollino.main import Cena, STYLE, Elemento, Sala as SalaVit, Labirinto, NADA, J, INVENTARIO as INV
ImportError: cannot import name 'J'

ImportError
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 268
    action()
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 2
    from _spy.vitollino.main import Cena, STYLE, Elemento, Sala as SalaVit, Labirinto, NADA, J, INVENTARIO as INV
'''},
{'date': 'Wed Feb 09 2022 21:48:11.39 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 3
    from _spy.vitollino.main import J
ImportError: cannot import name 'J'

ImportError
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 268
    action()
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 3
    from _spy.vitollino.main import J
'''},